### PR TITLE
DOCS-7681 / SE-13759 Calling Oracle Stored Procedure with TO_DATE() by DB

### PR DESCRIPTION
**Pull request by kevintroller**
Created new PR to avoid merging issues with multiple doc files.
Bringing commit [59a4c4c](https://github.com/mulesoft/docs-connectors/commit/59a4c4c4a212175fa1612625d933c9901ebd1c9a) changes from https://github.com/mulesoft/docs-connectors/pull/508

### CODE REVIEW

**Review by estebanwasinger — APPROVED** [review]

File: modules/ROOT/pages/db/database-connector-examples.adoc [hunk 1]
```diff
@@ -518,6 +518,50 @@ After execution, the resulting values of the `output` and `input-output` paramet
 
 For Database Connector version 1.4.0 or later, make sure that no whitespace is between the name of the stored procedure and the first parentheses.
 
+Database Connector supports the use of callable statements with the format ``{ call procedureName(:param1, :param2, ..., :paramN) }`` where parameters matches positionally, that is to say, `:paramN` matches the `Nth` parameter in the stored procedure declaration.
```
Comment (kbostrom) [inline modules/ROOT/pages/db/database-connector-examples.adoc:521]: You have already introduced this as "Anypoint Connector for Database (Database Connector)"?

Comment (kbostrom) [reply modules/ROOT/pages/db/database-connector-examples.adoc:521]:
```suggestion
Database Connector supports the use of callable statements with the format ``{ call procedureName(:param1, :param2, ..., :paramN) }``, in which the parameter number matches its position in the stored procedure declaration.
```
I don't think that you have to explain paramN. Maybe just end the example after ...,?

File: modules/ROOT/pages/db/database-connector-examples.adoc [hunk 2]
```diff
@@ -518,6 +518,50 @@ After execution, the resulting values of the `output` and `input-output` paramet
...
+
+Suppose you are using an Oracle database which you have initialized using with the following table and stored procedure:
```
Comment (kbostrom) [inline modules/ROOT/pages/db/database-connector-examples.adoc:523]: Clarify the "using" / "with" wording by choosing one that matches your intent:
Was the database initialized by using the table and procedure?
Or was the database initialized using the values in the table and the procedure?
Or was the database initialized to include the values in the table and the procedure?

File: modules/ROOT/pages/db/database-connector-examples.adoc [hunk 3]
```diff
@@ -518,6 +518,50 @@ After execution, the resulting values of the `output` and `input-output` paramet
...
+
+    CREATE PROCEDURE createEmployee(e_name VARCHAR2, e_age NUMBER, e_birth_date DATE) AS
+    BEGIN
+        INSERT INTO SYSTEM.employees(employee_name, employee_age, employee_birthday) VALUES(e_name, e_age, e_birth_date);
+    END;
+----
+
+The connector does not support named parameters in callable statements, so attempting to match parameters by name ( ``{ call createEmployee(employee_age => :age, e_birth_date => :date, e_name => :name) }`` ) does not work. Parameters must be provided in the appropriate order: `{ call createEmployee(e_name => :name, employee_age => :age, e_birth_date => :date) }`.
+
+[[use_dates_stored_procedures]]
```
Comment (kbostrom) [inline modules/ROOT/pages/db/database-connector-examples.adoc:544]: I have not seen this formatting; is it okay?

File: modules/ROOT/pages/db/database-connector-examples.adoc [hunk 4]
```diff
@@ -518,6 +518,50 @@ After execution, the resulting values of the `output` and `input-output` paramet
...
+== Use Dates on Stored Procedures
+Database Connector does not support the use of engine specific embedded functions, for example, if you want to change a date to a specific format before calling a procedure in an Oracle database, the following approach does not work:  `{ call createEmployee(e_name => :name, employee_age => :age, e_birth_date => TO_DATE(:date, 'YYYY-MM-DD HH:mm:ss')) }.` +
```
Comment (kbostrom) [inline modules/ROOT/pages/db/database-connector-examples.adoc]:
```suggestion
Database Connector does not support the use of engine-specific embedded functions. For example, if you want to change a date to a specific format before calling a procedure in an Oracle database, the following approach does not work:  
+
+ `{ call createEmployee(e_name => :name, employee_age => :age, e_birth_date => TO_DATE(:date, 'YYYY-MM-DD HH:mm:ss')) }.` +
```

File: modules/ROOT/pages/db/database-connector-examples.adoc [hunk 5]
```diff
@@ -518,6 +518,50 @@ After execution, the resulting values of the `output` and `input-output` paramet
...
+    );
+
+
+    CREATE PROCEDURE createEmployee(e_name VARCHAR2, e_age NUMBER, e_birth_date DATE) AS
+    BEGIN
+        INSERT INTO SYSTEM.employees(employee_name, employee_age, employee_birthday) VALUES(e_name, e_age, e_birth_date);
+    END;
+----
+
+The connector does not support named parameters in callable statements, so attempting to match parameters by name ( ``{ call createEmployee(employee_age => :age, e_birth_date => :date, e_name => :name) }`` ) does not work. Parameters must be provided in the appropriate order: `{ call createEmployee(e_name => :name, employee_age => :age, e_birth_date => :date) }`.
```
Comment (kbostrom) [inline modules/ROOT/pages/db/database-connector-examples.adoc:542]:
```suggestion
The connector does not support named parameters in callable statements, so attempting to match parameters by name ( ``{ call createEmployee(employee_age => :age, e_birth_date => :date, e_name => :name) }`` ) does not work. Parameters must be provided in the appropriate order: 
+
+ `{ call createEmployee(e_name => :name, employee_age => :age, e_birth_date => :date) }`.
```

File: modules/ROOT/pages/db/database-connector-examples.adoc [hunk 6]
```diff
@@ -518,6 +518,50 @@ After execution, the resulting values of the `output` and `input-output` paramet
...
+
+[[use_dates_stored_procedures]]
+== Use Dates on Stored Procedures
+Database Connector does not support the use of engine specific embedded functions, for example, if you want to change a date to a specific format before calling a procedure in an Oracle database, the following approach does not work:  `{ call createEmployee(e_name => :name, employee_age => :age, e_birth_date => TO_DATE(:date, 'YYYY-MM-DD HH:mm:ss')) }.` +
+Use DataWeave for all data transformations before the invocation to the callable statement, for the previous example we perform the following transformation:
```
Comment (kbostrom) [inline modules/ROOT/pages/db/database-connector-examples.adoc]:
```suggestion
Use DataWeave for all data transformations before the invocation to the callable statement. Using the previous example, for instance, perform the following transformation:
```

**Review by kbostrom — APPROVED** [review]
Edited

**Pull request merged**